### PR TITLE
[10.0][IMP] mis_builder_budget: Extend budget items w/ analytic tags

### PR DESCRIPTION
**Pull request by ghost**
Companion for v11: #101

### CODE REVIEW

**Review by tschanzt — APPROVED** [review]

**Comment by fclementic2c** [comment]
:+1:

**Review by sbidoul — APPROVED** [review]
LGTM. Please do not bump version in this PR. I bump after merging when releasing.

File: mis_builder_budget/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     'name': 'MIS Builder Budget',
```
Comment (simahawk) [inline mis_builder_budget/__manifest__.py]: no extra dependency? Or `analytic` comes from `mis_builder`? (I'd add it anyway to make it explicit)

**Review by simahawk — APPROVED** [review]

File: mis_builder_budget/__manifest__.py (hunk 1 above)
Comment (sbidoul) [reply mis_builder_budget/__manifest__.py]: we could add the dependency on `account` to be explicit, but mis_builder pulls it anyway

Comment (ghost) [reply mis_builder_budget/__manifest__.py]: @simahawk yup, this module depends on `analytic` indirectly:
`this_module` -> `mis_builder` -> `account` -> `analytic`

**Review by grindtildeath — APPROVED** [review]

**Comment by sbidoul** [comment]
I'll merge and relase soon.

**Pull request merged**